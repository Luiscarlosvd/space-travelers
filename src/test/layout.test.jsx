import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Layout from '../components/Layout';
import { expect, test } from 'vitest'

test('Layout renders correctly', () => {
  const { component } = render(
    <BrowserRouter>
      <Layout />
    </BrowserRouter>,
  );
  expect(component).toMatchSnapshot();
});