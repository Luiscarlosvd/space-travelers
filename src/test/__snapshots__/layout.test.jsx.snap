// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Layout renders correctly 1`] = `undefined`;
